apis/air_quality/src/lib.rs

```diff
@@ -0,0 +1,92 @@
+#![no_std]
+
+use core::cell::Cell;
+use libtock_platform::{
+    share::scope, share::Handle, DefaultConfig, ErrorCode, Subscribe, Syscalls,
+};
+use ReadType::{ReadCO2, ReadTVOC};
+
+#[cfg(test)]
+mod tests;
+
+enum ReadType {
+    ReadCO2 = READ_CO2 as isize,
+    ReadTVOC = READ_TVOC as isize,
```

Comment:
```suggestion
    CO2 = READ_CO2 as isize,
    TVOC = READ_TVOC as isize,
```

Reply:
I renamed the `ReadTVOC` value to `Tvoc` to avoid a warning. If needed, I will rename it to `TVOC`.